### PR TITLE
Add and update Chinese simplified translation

### PR DESCRIPTION
**Pull request by cloudhzc**
## 起飛前確認事項

- [√] 我已詳細確認編輯之文件有符合 [CONTRIBUTING.md](https://github.com/yeecord/docs/blob/master/CONTRIBUTING.md) 文件規範

https://github.com/yeecord/docs/pull/63
https://github.com/yeecord/docs/pull/62
https://github.com/yeecord/docs/pull/61
https://github.com/yeecord/docs/pull/60
https://github.com/yeecord/docs/pull/59
https://github.com/yeecord/docs/pull/55
前車之鑒

### CODE REVIEW

**Comment by cloudhzc** [comment]
Okay?

**Review by cloudhzc — COMMENTED** [review]
Oh

**Review by cloudhzc — COMMENTED** [review]
.

**Review by wolf-yuan-6115 — APPROVED** [review]

**Comment by wolf-yuan-6115** [comment]
?

**Comment by cloudhzc** [comment]
按錯鍵

File: docs/credit.mdx [hunk 1]
```diff
@@ -47,6 +47,10 @@ import CreditItem from "../src/components/CreditItem";
 - Wolf yuan#6173
 - MONEY#3897
 
+## 翻譯
+
+- Cloudhzc#2121
+
```
[the remark this answers is not in the document]
Comment (Gary50613) [reply docs/credit.mdx]: hmmmmmmmmmmmmmmm maybe yes

**Pull request merged**